custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -307,6 +307,248 @@
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "time_birth",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "time_birth"
+      },
+      {
+        "column_id": "father_name",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "father_name"
+      },
+      {
+        "column_id": "mother_name",
+        "datatype": "string",
+        "type": "raw",
+        "property_name": "mother_name"
+      },
+      {
+        "column_id": "private_admit",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "private_admit"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "date_last_private_admit",
+        "datatype": "date",
+        "type": "raw",
+        "property_name": "date_last_private_admit"
+      },
+      {
+        "column_id": "date_return_private",
+        "datatype": "date",
+        "type": "raw",
+        "property_name": "date_return_private"
+      },
+      {
+        "column_id": "primary_admit",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "primary_admit"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "is_oos",
+        "datatype": "small_integer",
+        "type": "expression",
+        "expression": {
+          "type": "switch",
+          "switch_on": {
+            "type": "property_name",
+            "property_name": "is_oos"
+          },
+          "cases": {
+            "no": {
+              "type": "constant",
+              "constant": 0
+            },
+            "yes": {
+              "type": "constant",
+              "constant": 1
+            }
+          },
+          "default": {
+            "type": "constant",
+            "constant": null
+          }
+        }
+      },
+      {
+        "column_id": "last_class_attended_ever",
+        "datatype": "integer",
+        "type": "raw",
+        "property_name": "last_class_attended_ever"
+      },
+      {
```

Comment:
This is also ccs record case property that is already stored in the ccs record UCR, and mostly only makes sense in the context of active pregnancies. What will this be used for? I could not find it in the spec, but if it is in the pregnancy part of the API, the ccs record one is probably fine

Reply:
Actually this not mentioned in any category in the first tab of the sheet. Not sure, where will this fit.
is ccs one  preg_order same as person case one?

Reply:
I have asked Zohaib for confirmation. But it sounds like they should be same. If Zohaib does not respond by the EOD, I will remove ti.

Reply:
yes, he confirmed that we can use ccs one. removing this